lib/create-space-api.js

```diff
@@ -206,7 +206,8 @@ import omit from 'lodash/omit'
  * @return {ContentfulSpaceAPI}
  */
 export default function createSpaceApi ({
-  http
+  http,
+  httpUploads
```

Comment:
That was the only way I could find to nicely use two API endpoints. Since the httpClient has to know the Space ID and I cant simply overwrite the host in the Upload related HTTP requests via the axios request config parameter.